reference/src/client/include/sa_ta_types.h

```diff
@@ -50,6 +50,7 @@ typedef enum {
     SA_KEY_GENERATE,
     SA_KEY_EXPORT,
     SA_KEY_IMPORT,
+    SA_KEY_PROVISION,
```

Comment:
Agree, but that also requires that there is no code that is written that implies or expects a value to be consistently the same. Many applications that use enums usually append the next item to the end of the list to avoid potential coding pitfalls.